steamtinkerlaunch

```diff
@@ -18243,6 +18249,11 @@ function checkIntDeps {
 		if [ -z "$YAD" ]; then
 			YAD="$(command -v "yad")"
 		fi
+		
+		if [ ! "$YAD" ]; then
+			writelog "ERROR" "${FUNCNAME[0]} - Could not find yad" "E"
```

Comment:
I know we're probably going to remove this but just for future note, `writelog` usage is like this:

```suggestion
			writelog "ERROR" "${FUNCNAME[0]} - Could not find yad"
```